db/migrate/20240530200000_add_deletion_date_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeletionDateToUser < ActiveRecord::Migration[7.0]
```

Comment:
I am slightly confused here because we are on Rails 7.1 so the migration file should state `ActiveRecord::Migration[7.1]`.
Autogenerated migrations contain the correct version, I checked that.